windows.versions

```diff
@@ -0,0 +1,12 @@
+{
+  "windows": [
```

Comment:
@rawahars What is the purpose of this file? It tracks which releases have windows builds? What does the build number below mean? What would it mean for the build number to be 2 instead of 1?

Reply:
Hmmm how does this interact with the monthly tuesday updates? On that day do we need to update the build numbers or not?

Reply:
So you are right. Currently, we do not support that. This is just an addition which we may or may not use.
OS patches would happen anyways. If we want to never change the artifacts that is fine as well. We always keep the build number to 1.

I feel it might be better to have a failsafe here than just not have it altogether.